common/components/controllers/SectionController.jsx

```diff
@@ -132,6 +133,8 @@ class SectionController extends React.Component<{||}, State> {
         return <AboutEventProjectController />;
       case Section.CreateEventProject:
         return <CreateEventProjectController />;
+      case Section.Privacy:
+        return <PrivacyController />;
```

Comment:
This switch statement handles clientside routing. Section.Privacy is managed by enums/Section.js, and PrivacyController is the export from PrivacyController.jsx.